teku/src/main/java/tech/pegasys/teku/cli/options/Eth2NetworkOptions.java

```diff
@@ -93,6 +95,15 @@
       arity = "1")
   private UInt64 terminalBlockHashEpochOverride;
 
+  @Option(
+      names = {"--Xnetwork-safe-slots-to-import-optimistically"},
+      hidden = true,
+      paramLabel = "<NUMBER>",
+      description =
+          "Override the the number of slots that must pass before it is considered safe to optimistically import a block",
+      arity = "1")
+  private Integer safeSlotsToImportOptimistically = DEFAULT_SAFE_SLOTS_TO_IMPORT_OPTIMISTICALLY;
```

Comment:
Given this is an integer, we might want to put validation around it not being negative, and maybe also build in an upper limit of what might be sane?

Reply:
I don't think there is a sane upper limit - a big number essentially just disables optimistic sync which is potentially useful.
Added check it's not negative.